subworkflows/local/longread_adaterremoval.nf

```diff
@@ -0,0 +1,29 @@
+//
```

Comment:
This file name has a typo, shoudl be `longread_adapterremoval` (you're missing the `p`)